### ISSUE TITLE
Question about results on DBLP

### ISSUE THREAD

**Issue by PlusRoss** [malllabiisc/HyperGCN#1]
Hi,

Thanks for the excellent work. I see that you've released the DBLP data. When I directly run hypergcn.py (by setting _mediators = True_), however, the test error I obtained is around 30, which does not match the results in the paper (Table 4). When I change the learning rate to 0.1, the test error can be reduced to about 13, but this is still not the test error reported in the paper.  Did I make any mistake?

**Comment by naganandy** [comment]
Hi,

Thanks for the interest in our work. The results reported in Table 4 are mean and standard deviation over different train-test splits with learning rate = 0.01. It is an interesting observation that the error can be (significantly) reduced by changing the learning rate!

**Comment by PlusRoss** [comment]
Thanks for your reply! I'm still confused about the train-test split. Do you mean that all the nodes are split into training or test set without any exception? If so, I notice that in the co-citation dataset, there exists a large portion of nodes (e.g. 40% in Cora) that do not appear in any hyperedge. Are they also included in the training or test set?

**Comment by naganandy** [comment]
yes, they are included without exception. GCN uses node's features (e.g. bag-of-words) even when the node does not appear in any hyperedge.

**Comment by PlusRoss** [comment]
Thanks for your answer!
